Userland/Libraries/LibWeb/Cookie/Cookie.cpp

```diff
@@ -8,6 +8,21 @@
 #include <LibIPC/Decoder.h>
 #include <LibIPC/Encoder.h>
 
+StringView Web::Cookie::same_site_mode_to_string(Web::Cookie::SameSiteMode same_site_mode)
```

Comment:
Please place this into a `namespace` instead:

```cpp
namespace Web::Cookie {

StringView same_site_mode_to_string(Web::Cookie::SameSiteMode same_site_mode)
{
    // ...
}

}
```

Reply:
Thank you, I have pushed new chages